manifest/proxy.go

```diff
@@ -0,0 +1,42 @@
+package manifest
+
+import "fmt"
+
+type Proxy struct {
+	Name string
+
+	Balancer  int
+	Container int
+
+	Protocol string
+	Host     string
+	Proxy    bool
+	Secure   bool
+}
+
+func (p *Proxy) Start() error {
```

Comment:
I've observed the proxy container hanging around on `convox start` crashes. Can we address this with a defer or something?